edgexfoundry/go-mod-core-contracts
# feat: Remove JSON omitempty from SimpleReading value

The SimpleReading can contain the empty string.

Close #725







If your build fails due to your commit message not passing the build checks, please review the guidelines here: https://github.com/edgexfoundry/go-mod-core-contracts/blob/main/.github/Contributing.md

## PR Checklist
Please check if your PR fulfills the following requirements:

- [ ] I have added unit tests for the new feature or bug fix (if not, why?) **not impact the test**
- [ ] I have opened a PR for the related docs change (if not, why?) **not impact doc**
  <link to docs PR>

## Testing Instructions
- Run edgex and device service
- Query empty string value from device service
- Inspect the reading should contain empty string value

## New Dependency Instructions (If applicable)
